metagraph-plugin/conda_recipe/meta.yaml

```diff
@@ -29,6 +32,14 @@ test:
   imports:
     - metagraph
     - katana
+    - pytest
+  requires:
+    - pytest
+  source_files:
+    - tests
+  script:
+    - run_pytest.sh
```

Comment:
NIT: Since the script is so simple you can just use `commands`. https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands